### ISSUE TITLE
Multi line use statements are fixed wrong by ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName

### ISSUE THREAD

**Issue by klausi** [slevomat/coding-standard#1581]
Use statements can be grouped and span multiple lines, example:
```php
<?php

/**
 * @file
 * Example.
 */

use Test\MultiLine as MultiLineAlias,
    Test\MultiLineSecond;

$a = new MultiLineAlias();
$b = new MultiLineSecond();
```
Reported error: Constant Test\MultiLineSecond should not be referenced via a fully qualified name, but via a use statement. (SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName)

That is misleading, no constant is defined here.

Furthermore the file is fixed wrong and results in broken code:
```php
<?php

/**
 * @file
 * Example.
 */

$a = new MultiLineAlias();
$b = new \MultiLineSecond();
```
The use statement is completely deleted.

I saw that there was some discussion that Slevomat does not support multiline use statements in #1547 and #1554, but then the fixer should at least ignore those and not generate broken PHP files.

Background: trying to replace Drupal Coder's custom use statement sniff with your upstream code to avoid maintaining it in https://github.com/pfrenssen/coder/pull/199

**Comment by kukulich** [comment]
Yes, we don't support group uses.

**Comment by jonathan1055** [comment]
@klausi said
> the fixer should at least ignore those and not generate broken PHP files.

Does the fixer still generate broken PHP?

**Comment by kukulich** [comment]
> Does the fixer still generate broken PHP?

Yes, because we don't have support for parsing group uses.

**Comment by jonathan1055** [comment]
The fixer could be prevented from breaking the php by simply detecting if there is a group use, and then not doing any fixing, just report the error. I did this in https://github.com/slevomat/coding-standard/pull/1555
I could investigate to see if this could be done here too

**Comment by kukulich** [comment]
Yes, it could be solved this way.
